docker/compose.xml

```diff
@@ -13,362 +13,398 @@
       <manifest-digest sha256new="YS7DN65H44DWTAUYSM63GESSEM52PHQLEHIZTTV2253G2QIELGPQ"/>
       <file dest="docker-compose.exe" href="https://github.com/docker/compose/releases/download/1.10.0-rc1/docker-compose-Windows-x86_64.exe" size="6340246"/>
     </implementation>
-    <implementation arch="Linux-x86_64" id="sha1new=2dade4e6092d6341c55b033723ac9276792f9e06" if-0install-version="2.14.1.." main="docker-compose" released="2017-01-04" stability="testing" version="1.10.0-rc1">
```

Comment:
Why is this version constraint being removed? (I don't know what it was for, but I assume it was important for something...)

Reply:
The Linux and Mac builds require the new `<file executable='true' ... />` feature we talked about on the mailing list a while ago (basically running `chmod +x` after the download).

I already implemented this in the .NET version of Zero Install and am using that via .NET Core on Linux. However, since it's not yet supported in the OCaml version I realized that I don't really have any sensible value for the `if-0install-version` constraint (yet).